README.md

```diff
@@ -53,11 +53,17 @@ See our docs for more info on how to use the library.
 
 ### Examples
 
-To run an example, clone the repo and run the following commands
-
+To run an example, clone the repo and follow these steps
+1. Copy contents of `/apps/web/.env.example` into a new file, `/apps/web/.env`
+2. Copy contents of `/examples/.env.example` into a new file, `/examples/.env.example` and add your [OpenAI API Key](https://openai.com/api/) to the `OPENAI_API_KEY` field
```

Comment:
```suggestion
2. Copy contents of `/examples/.env.example` into a new file, `/examples/.env` and add your [OpenAI API Key](https://openai.com/api/) to the `OPENAI_API_KEY` field
```

Reply:
thanks for catching that, committed you're change